PixiEditor.ChangeableDocument/Changes/Drawing/ShiftLayer_UpdateableChange.cs

```diff
@@ -14,10 +14,9 @@ public ShiftLayer_UpdateableChange(Guid layerGuid, VecI delta)
 
     public override OneOf<Success, Error> InitializeAndValidate(Document target)
     {
-        var member = target.FindMember(layerGuid);
-        if (member is not Layer)
-            return new Error();
-        return new Success();
+        if (target.HasMember(layerGuid))
```

Comment:
This is different from the old check, I'm specifically looking for a layer, not just for an existing member